### PR TITLE
Added debug logs

### PR DESCRIPTION
**Pull request by sudheerdulam**

### CODE REVIEW

File: app_manager.go [hunk 1]
```diff
@@ -20,6 +24,7 @@ func GetAppManager() *AppManager {
 
 func (manager *AppManager) RegistApplication(app interface{}) {
 	manager.applications = append(manager.applications, app)
+	fmt.Printf("Registered app %+v\n", app)
```
Comment (nutant-shobhit) [inline app_manager.go:27]: We should use alog instead of fmt

**Review by nutant-shobhit — CHANGES_REQUESTED** [review]
Can we change the logger to alog instead of fmt

File: app_manager.go (hunk 1 above)
Comment (alolage-nx) [reply app_manager.go:27]: let's not change logger in gofc. Its from open source code. we have logger setup in CNC which redirects stdout/stderr to CNC logs. That's how we could debug blocking issue.

**Review by alolage-nx — APPROVED** [review]
LGTM.

**Pull request merged**